### PR TITLE
[RG-372] Add vcd waveform files support for IPs

### PR DESCRIPTION
**Pull request by KochynVolodymyr**
### Motivate of the pull request
 - [x] To address an existing issue. If so, please provide a link to the issue: RG-372
 - [ ] Breaking new feature. If so, please describe details in the description part.

 ### Describe the technical details
Add support of vcd file extension for IP simulation waveform.
image

 ### Which part of the code base require a change
 
 - [x] Library: ipgenerate, utils
 - [ ] Plug-in: <Specify the plugin name>
 - [ ] Engine
 - [ ] Documentation
 - [ ] Regression tests
 - [ ] Continous Integration (CI) scripts

### CODE REVIEW

File: src/IPGenerate/IPGenerator.cpp [hunk 1]
```diff
@@ -637,13 +637,15 @@ std::pair<bool, std::string> IPGenerator::OpenWaveForm(
...
-  return {ok, "Command \'" + cmd + "\' failed."};
+  for (auto ext : {".fst", ".vcd"}) {
+    auto file = FileUtils::FindFileByExtension(artifactsPath, ext);
+    if (!file.empty()) {
+      const std::string cmd = "wave_open " + file.string();
+      bool ok = GlobalSession->CmdStack()->push_and_exec(new Command(cmd));
+      return {ok, "Command \'" + cmd + "\' failed."};
+    }
+  }
+  return {false, "Waveform file does not exist."};
```
Comment (ravic-rs) [inline src/IPGenerate/IPGenerator.cpp:648]: We should let users know the supported waveforms in failure.

**Review by ravic-rs — COMMENTED** [review]
1 comment to address

File: src/IPGenerate/IPGenerator.cpp (hunk 1 above)
Comment (KochynVolodymyr) [reply src/IPGenerate/IPGenerator.cpp:648]: Please suggest message text that will be better in this case

Comment (KochynVolodymyr) [reply src/IPGenerate/IPGenerator.cpp]: Done

**Pull request merged**